2_adventures/001/kotp/16_Readme_Again/README.md

```diff
@@ -0,0 +1,9 @@
+# Pair Programming Monday!
+
+Spent 3 hours pair programming today, covered a wide variety of technologies.
+
+Spent some time rebasing commits.  The workflow for the challenge means that we
```

Comment:
yep! exactly what I talk about in the [suggested git workflow](https://github.com/RubySteps/21-day-challenge/wiki/Suggested-Git-Workflow)